va/va_enc_vp8.h

```diff
@@ -128,7 +128,20 @@ typedef struct  _VAEncPictureParameterBufferVP8
             unsigned int no_ref_gf                      : 1;
             /* don't reference the alternate reference frame */
             unsigned int no_ref_arf                     : 1;
-            unsigned int reserved                       : 28;
+            /* The temporal id the frame belongs to. */
+            unsigned int temporal_id                    : 8;
+            /**
+            *  following two flags indicate the reference order
```

Comment:
Is 0 a valid value for first_ref/second_ref?People usually memset the structure and set some value need by them. If 0 is not a valid value for them. It may introduce inconvenience.

Reply:
temporary remove them

Reply:
it can be 0, 0 means it will take no effect